### ISSUE TITLE
Docker Hub Tagging

### ISSUE THREAD

**Issue by chaseconey** [nimmis/docker-apache-php7#3]
Is there anyway you could start adding tags to the Docker Hub releases? Would love to pin to a version for production use.

**Comment by nimmis** [comment]
you mean the php version i assume, i'll check if it is possible to build on specific version (prob srpm builds then)

**Comment by chaseconey** [comment]
I mean the actual Docker images that you ship, i.e. [this tab](https://hub.docker.com/r/nimmis/apache-php7/tags/).

This will allow you to iterate without potentially breaking people that rely on a certain setup at a point in time.

**Comment by nimmis** [comment]
Yes I understood that, but I assumed that you wanted it to be tagged with the php-version as the rebuild is done automatic when the OS is updated.
